cmake/external_nnp_transformer.cmake

```diff
@@ -16,38 +16,38 @@
 
 # To build ngraph with Argon transformer using pre-build Argon API
 # ```
-# cmake -DNGRAPH_ARGON_ENABLE=True -DNGRAPH_PREBUILD_ARGON_API_PATH=$HOME/dev/system/_out/debug-x86_64-Linux ..
+# cmake -DNGRAPH_NNP_ENABLE=True -DNGRAPH_PREBUILD_ARGON_API_PATH=$HOME/dev/system/_out/debug-x86_64-Linux ..
 # make -j
 # ```
 
 # Enable ExternalProject CMake module
 include(ExternalProject)
 
-if (NGRAPH_ARGON_ENABLE)
+if (NGRAPH_NNP_ENABLE)
     # We require pre-build Argon API library
     if (NOT DEFINED NGRAPH_PREBUILD_ARGON_API_PATH)
         message(FATAL_ERROR "NGRAPH_PREBUILD_ARGON_API_PATH not defined, set it with -DNGRAPH_PREBUILD_ARGON_API_PATH=")
     endif()
 
     # Repository
-    if (DEFINED CUSTOM_ARGON_TRANSFORMER_GIT_REPOSITORY)
-        set(ARGON_TRANSFORMER_GIT_REPOSITORY ${CUSTOM_ARGON_TRANSFORMER_GIT_REPOSITORY})
+    if (DEFINED CUSTOM_NNP_TRANSFORMER_GIT_REPOSITORY)
+        set(NNP_TRANSFORMER_GIT_REPOSITORY ${CUSTOM_NNP_TRANSFORMER_GIT_REPOSITORY})
     else()
-        set(ARGON_TRANSFORMER_GIT_REPOSITORY git@github.com:NervanaSystems/argon-transformer.git)
+        set(NNP_TRANSFORMER_GIT_REPOSITORY git@github.com:NervanaSystems/argon-transformer.git)
     endif()
 
-    # Set argon_transformer tag
+    # Set nnp_transformer tag
     # Notes:
-    # - Before we have ngraph CI job for argon transformer, ngraph master might not be
-    #   compatible with argon transformer. To ensure compatibility, checkout the ngraph commit point
-    #   where the following `ARGON_TRANSFORMER_GIT_TAG` is set and build ngraph with argon using this
+    # - Before we have ngraph CI job for nnp transformer, ngraph master might not be
+    #   compatible with nnp transformer. To ensure compatibility, checkout the ngraph commit point
+    #   where the following `NNP_TRANSFORMER_GIT_TAG` is set and build ngraph with nnp using this
     #   commit.
-    # - After we have ngraph CI job for argon transformer, ngraph master will be compatible with
-    #   argon transformer guaranteed by CI.
-    set(ARGON_TRANSFORMER_GIT_TAG cpp-master)
+    # - After we have ngraph CI job for nnp transformer, ngraph master will be compatible with
+    #   nnp transformer guaranteed by CI.
+    set(NNP_TRANSFORMER_GIT_TAG sandeep/rename-nnp)
```

Comment:
We shall keep it as `cpp-master`, and `git checkout` locally if we want to test other branches.